src/main/java/data/Profession.java

```diff
@@ -9,12 +9,12 @@
     private final WorkingArea workingArea;
     private final TimeSpan maxWorkingTime;
     private final double wage;
-    
+
     /**
      * Constructs a new {@link Profession profession} instance.
      * @param departmentName - Name of the department this profession is associated with.
      * @param workingArea - {@link WorkingArea} of this profession.
-     * @param maxWorkingHours - Maximum number of hours an {@link Employee employee} may work per month.
+     * @param maxWorkingTime - Maximum number of hours an {@link Employee employee} may work per month.
```

Comment:
I agree with @larsk21 that the description should also be changed in order to avoid another PR concerning the same documentation issue.